pwiz_tools/Skyline/Model/AuditLog/AuditLogEntry.cs

```diff
@@ -1457,8 +1510,12 @@ public byte[] GetAuditLogHash()
                 encodedBytes.Add(enc.GetBytes(SkylineVersion));
                 encodedBytes.Add(GetTimeStampBytesForHash());
                 encodedBytes.Add(enc.GetBytes(TimeZoneOffset.TotalHours.ToString(CultureInfo.InvariantCulture)));
-
-                // Avoid heap thrash performance issue by carefully allocating hash buffer size
+                if (Parent != null && !Parent.IsRoot && docFormat >= DocumentFormat.SEQUENTIAL_LOG_HASH)
+                {
+                    encodedBytes.Add(Convert.FromBase64String(Parent.Hash.ActualHash.HashString));
```

Comment:
This line causes a StackOverflowException in TestBlibMoleculeLists in the 32 bit build.

Reply:
Fix committed.